### DISCUSSION TITLE
Updates request

### DISCUSSION THREAD

**Discussion by MiMillieuh** [MiMillieuh/AxieInfinityOriginsLinux#1] [Ideas]
If I miss an update you can notice it here.

Please be aware that I might take some time to update and also if someone already asked for an update please don't ask again.

**Comment by duanxyz** [comment]
for version 0.2.35 sir

**Reply by MiMillieuh** [reply]
Should be coming in the next hour sir :)

**Reply by MiMillieuh** [reply]
Updated RN
